satpy/etc/readers/nc_slstr.yaml

```diff
@@ -181,31 +181,35 @@ datasets:
     coordinates: [longitude, latitude]
     file_type: esa_l1b_tir
 
-
+    
   solar_zenith_angle:
     name: solar_zenith_angle
     sensor: slstr
     resolution: 1000
     coordinates: [longitude, latitude]
     file_type: esa_angles
+    file_key: solar_zenith_tn
```

Comment:
This fixes the variable to the nadir view only, is that what we want? otherwise we could have something like `file_key: solar_zenith_t{view}` for example?